test/math.test.ts

```diff
@@ -0,0 +1,13 @@
+import { clamp } from 'math';
```

Comment:
We generally prefer to use colocated tests, rather than a separate `src` and `tests` folder. Something like this:

```
src/
  math/
    tests/
      math.test.ts
    math.ts
    index.ts
  dom/
    tests/
      dom.test.ts
    dom.ts
    index.ts
  ...
```

The folder structure is a bit more verbose for creating utilities, but it is quite pleasant for working with existing utilities.

I wonder if we would like to keep that colocation structure in this case, though, since `javascript-utilities` is pretty flat, compared to repos like `polaris`.

@kaelig what are your thoughts?

Reply:
Thanks for asking ❤️ Agreed on colocating tests. 

On Polaris-styleguide we colocate tests at the same level as the modules themselves (without a “test” directory), I guess you could also do something like that: https://github.com/Shopify/polaris-styleguide/tree/master/server/styleguide-schema/content-parsers